### PR TITLE
Final version of the New York Times clone

### PR DESCRIPTION
**Pull request by kotoku-code**
## Positioning and floating elements:
- Added a Nav-bar with float
- Added a Main with float
- Added a Footer with float
[Live Demo Link](https://raw.githack.com/Spayco/NYT-Clone/beta/index.html)

### CODE REVIEW

**Comment by kotoku-code** [comment]
checked stickler locally through vscode

File: index.html [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+            <button class="btn">LOG IN</button>
+        </div>
+    </nav>
+    <main>
+        <h1 id="title">Space Ripples Reveal <br> Big Bang’s Smoking Gun</h1>
+        <figure>
+        <img class="title-img" src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-superJumbo.jpg?quality=90&auto=webp" alt="alan guth beside a telescope">
+        <figcaption>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit...<p>Rick Friedman for The New York Times</p></figcaption>
+    </figure>
+    <!-- TODO: links in articles -->
```
Comment (euqueme) [inline index.html]: Remove comment in line #31

File: index.html [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times</title>
+    <link rel="stylesheet" href="https://g1.nyt.com/fonts/css/web-fonts.5810def60210a2fa7d0848f37e3fa048bb6147b1.css" type="text/css">
+
+    <link rel="stylesheet" href="./styles.css" type="text/css">
+</head>
+<body>
+    <nav id="nav">
+        <div class="nav-element left-nav-element">
+            <button class="nav-btn" ><svg viewBox="0 0 16 16"><rect x="1" y="3" fill="#333333" width="14" height="2"></rect><rect x="1" y="7" fill="#333333" width="14" height="2"></rect><rect x="1" y="11" fill="#333333" width="14" height="2"></rect></svg></button>
+            <button class="nav-btn"><svg viewBox="0 0 16 16"><path fill="#333" d="M11.3,9.2C11.7,8.4,12,7.5,12,6.5C12,3.5,9.5,1,6.5,1S1,3.5,1,6.5S3.5,12,6.5,12c1,0,1.9-0.3,2.7-0.7l3.3,3.3c0.3,0.3,0.7,0.4,1.1,0.4s0.8-0.1,1.1-0.4c0.6-0.6,0.6-1.5,0-2.1L11.3,9.2zM6.5,10.3c-2.1,0-3.8-1.7-3.8-3.8c0-2.1,1.7-3.8,3.8-3.8c2.1,0,3.8,1.7,3.8,3.8C10.3,8.6,8.6,10.3,6.5,10.3z"></path></svg></button>
```
Comment (euqueme) [inline index.html:16]: All style selectors have to be specified in your CSS file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
... 1.3v.1l1.5 1.3V16c-.8.5-1.7.6-2.5.6-1.5 0-2.8-.4-3.9-1.6l4.1-2V6l-5 2.2c.5-1.3 1.6-2.2 2.6-2.9l-.1-.2c-3 .8-5.7 3.5-5.7 6.9 0 4 3.3 7 7 7 4 0 6.6-3.2 6.6-6.5h-.2c-.6 1.3-1.5 2.5-2.6 3.1v-4.1l1.6-1.3v-.1L140 8.8v-3c1.5 0 3-1 3-2.9zm-8.7 11l-1.2.6c-.7-.9-1.1-2.1-1.1-3.8 0-.7.1-1.5.3-2.1l2.1-.9-.1 6.2zm12.2-12h-.1l-2 1.7v.1l1.7 1.9h.2l2-1.7v-.1l-1.8-1.9zm3 14.8l-.8.5-1-1V9.3l1-.7-.2-.2-.7.6-1.8-2.1-2.9 2 .2.3.7-.5.9 1.1v6.5l-1.3 1 .1.2.7-.5 2.2 2 3-2-.1-.3zm16.7-.1l-.7.5-1.1-1V9.3l1-.8-.2-.2-.8.7-2.3-2.1-3 2.1-2.3-2.1L154 9l-1.8-2.1-2.9 2 .1.3.7-.5 1 1.1v6.5l-.8.8 2.3 1.9 2.2-2-.9-.9V9.3l.9-.6 1.5 1.4v6l-.8.8 2.3 1.9 2.2-2-.9-.9V9.3l.8-.5 1.6 1.4v6l-.7.7 2.3 2.1 3.1-2.1v-.3zm8.7-1.5l-2.5 1.9-2.5-2v-1.2l4.7-3.2v-.1l-2.4-3.6-5.2 2.8v6.8l3.5 2.5 4.5-3.6-.1-.3zm-5-1.7V8.5l.2-.1 2.2 3.5-2.4 1.5zm14.1-.9l-1.9-1.5c1.3-1.1 1.8-2.6 1.8-3.6v-.6h-.2c-.2.5-.6 1-1.4 1-.8 0-1.3-.4-1.8-1L176 9.3v3.6l1.7 1.3c-1.7 1.5-2 2.5-2 3.3 0 1 .5 1.7 1.3 2l.1-.2c-.2-.2-.4-.3-.4-.8 0-.3.4-.8 1.2-.8 1 0 1.6.7 1.9 1l4.3-2.6v-3.6h-.1zm-1.1-3c-.7 1.2-2.2 2.4-3.1 3l-1.1-.9V8.1c.4 1 1.5 1.8 2.6 1.8.7 0 1.1-.1 1.6-.4zm-1.7 8c-.5-1.1-1.7-1.9-2.9-1.9-.3 0-1.1 0-1.9.5.5-.8 1.8-2.2 3.5-3.2l1.2 1 .1 3.6z"></path></svg></a>
```
Comment (euqueme) [inline index.html:19]: All styling selectors have to be specified in your CSS file

File: index.html [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+        <button class="circle-icons"><svg viewBox="0 0 15 9" class="css-4brsb6" width="15" height="9"><path fill-rule="evenodd" clip-rule="evenodd" d="M.906 8.418V0L5.64 4.76.906 8.419zm13 0L9.174 4.761 13.906 0v8.418zM7.407 6.539l-1.13-1.137L.907 9h13l-5.37-3.598-1.13 1.137zM1.297 0h12.22l-6.11 5.095L1.297 0z" fill="#000"></path></svg></button>
+        <button class="circle-icons"><svg class="css-uhuo44" viewBox="0 0 16 13" width="16" height="13"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.406 5.359L8.978 0v3.215C3.82 3.215.406 8.107.406 12.66 1.653 9.133 4.29 7.517 8.978 7.517v3.2l6.428-5.358z" fill="#000"></path></svg></button>
+        <button id="bookmark"><svg stroke="none" width="16px" height="25px" viewBox="0 0 16 25" class="css-ubw31m"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g stroke="#666" fill="none"><path fill="none" d="M0.5,0.5 L0.5,24.5 L8.1716264,19.5137775 L15.5,24.4404006 L15.5,0.5 L0.5,0.5 Z"></path></g></g></svg></button>
```
Comment (euqueme) [inline index.html]: All style selectors have to be specified in your CSS file

File: index.html [hunk 5]
```diff
@@ -0,0 +1,432 @@
...
+            <h3><a   href="#">Schiff, Calling Trump &lsquo;Wrathful and Vindictive,&rsquo; Sees Tweet as a Threat</a></h3>
+            <h3><a   href="#">How John Mulaney, Comedian, Spends His Sundays</a></h3>
+            </div>
+            </div>
+    </div>
+    <div class="ad">advertisement</div>
+</main>
+<footer>
+<div class="bottom-section"><hr style="margin-left: 12%; margin-right: 12%; border: none; height: 1px; background-color: #333; margin-bottom: 1px;">
+    <hr style="margin-left: 12%; margin-right: 12%;">
```
Comment (euqueme) [inline index.html]: All style selectors have to be specified in your CSS file

File: index.html [hunk 6]
```diff
@@ -0,0 +1,432 @@
...
...-1.5 1.3v.1l1.5 1.3V16c-.8.5-1.7.6-2.5.6-1.5 0-2.8-.4-3.9-1.6l4.1-2V6l-5 2.2c.5-1.3 1.6-2.2 2.6-2.9l-.1-.2c-3 .8-5.7 3.5-5.7 6.9 0 4 3.3 7 7 7 4 0 6.6-3.2 6.6-6.5h-.2c-.6 1.3-1.5 2.5-2.6 3.1v-4.1l1.6-1.3v-.1L140 8.8v-3c1.5 0 3-1 3-2.9zm-8.7 11l-1.2.6c-.7-.9-1.1-2.1-1.1-3.8 0-.7.1-1.5.3-2.1l2.1-.9-.1 6.2zm12.2-12h-.1l-2 1.7v.1l1.7 1.9h.2l2-1.7v-.1l-1.8-1.9zm3 14.8l-.8.5-1-1V9.3l1-.7-.2-.2-.7.6-1.8-2.1-2.9 2 .2.3.7-.5.9 1.1v6.5l-1.3 1 .1.2.7-.5 2.2 2 3-2-.1-.3zm16.7-.1l-.7.5-1.1-1V9.3l1-.8-.2-.2-.8.7-2.3-2.1-3 2.1-2.3-2.1L154 9l-1.8-2.1-2.9 2 .1.3.7-.5 1 1.1v6.5l-.8.8 2.3 1.9 2.2-2-.9-.9V9.3l.9-.6 1.5 1.4v6l-.8.8 2.3 1.9 2.2-2-.9-.9V9.3l.8-.5 1.6 1.4v6l-.7.7 2.3 2.1 3.1-2.1v-.3zm8.7-1.5l-2.5 1.9-2.5-2v-1.2l4.7-3.2v-.1l-2.4-3.6-5.2 2.8v6.8l3.5 2.5 4.5-3.6-.1-.3zm-5-1.7V8.5l.2-.1 2.2 3.5-2.4 1.5zm14.1-.9l-1.9-1.5c1.3-1.1 1.8-2.6 1.8-3.6v-.6h-.2c-.2.5-.6 1-1.4 1-.8 0-1.3-.4-1.8-1L176 9.3v3.6l1.7 1.3c-1.7 1.5-2 2.5-2 3.3 0 1 .5 1.7 1.3 2l.1-.2c-.2-.2-.4-.3-.4-.8 0-.3.4-.8 1.2-.8 1 0 1.6.7 1.9 1l4.3-2.6v-3.6h-.1zm-1.1-3c-.7 1.2-2.2 2.4-3.1 3l-1.1-.9V8.1c.4 1 1.5 1.8 2.6 1.8.7 0 1.1-.1 1.6-.4zm-1.7 8c-.5-1.1-1.7-1.9-2.9-1.9-.3 0-1.1 0-1.9.5.5-.8 1.8-2.2 3.5-3.2l1.2 1 .1 3.6z"></path></svg>
```
Comment (euqueme) [inline index.html:328]: All style selectors have to be specified in your CSS file

File: index.html [hunk 7]
```diff
@@ -0,0 +1,432 @@
...
+        <a href="#">Go To Home Page »</a>
+        <div style="clear: both;"></div>
+    </div>
+    <div style="margin-left: 5%;">
```
Comment (euqueme) [inline index.html]: All style selectors have to be specified in your CSS file

File: index.html [hunk 8]
```diff
@@ -0,0 +1,432 @@
...
+            <li><a href="#"><h3><svg class="css-r5ic95" viewBox="0 0 13 13" fill="#000"><path d="M12,2.9L9.6,5.2c-0.1,0.1-0.3,0.1-0.4,0C9.1,5.2,9.1,5,9.3,4.9l2.4-2.4c-0.2-0.2-0.3-0.3-0.5-0.5L8.7,4.3c-0.1,0.1-0.3,0.1-0.4,0C8.2,4.3,8.2,4.1,8.4,4l2.4-2.4c-0.3-0.3-0.5-0.5-0.5-0.5L7.6,3.4C7.1,4,6.8,5.1,7.1,5.8c-1.4,1-4.6,3.5-5.1,4c-0.8,0.8-0.4,1.8-0.3,1.9c0,0,0,0,0,0c0,0,0,0,0,0c0.1,0.1,1.1,0.5,1.9-0.3c0.4-0.4,2.9-3.6,3.9-5C8.4,6.9,9.6,6.6,10.2,6l2.3-2.6C12.5,3.4,12.3,3.2,12,2.9z"></path><path d="M0.8,1.9l0.3-0.3c0.9-0.9,3.2,1.1,3.8,1.7s0.9,1.8,0.4,2.6c1.4,1.1,4.6,3.5,5,3.9c0.8,0.8,0.4,1.8,0.3,1.9c0,0,0,0,0,0c0,0,0,0,0,0c-0.1,0.1-1.1,0.5-1.9-0.3c-0.4-0.4-2.9-3.7-4-5.1C3.9,6.7,2.9,6.4,2.3,5.8S-0.2,2.9,0.8,1.9z"></path></svg>Cooking</h3></a></li>
```
Comment (euqueme) [inline index.html]: All style selectors have to be specified in your CSS file

File: index.html [hunk 9]
```diff
@@ -0,0 +1,432 @@
...
+        <a href="#">Go To Home Page »</a>
+        <div style="clear: both;"></div>
```
Comment (euqueme) [inline index.html]: All style selectors have to be specified in your CSS file

File: fonts.css [hunk 10]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-cheltenham';
+  src:
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-normal-800.78ace43907be7e8789003cfd692c3ba4.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-normal-800.3759c9080a568598a298f0b0bc5a5d12.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-normal-800.beea606b16c50bd15a1851fba1dd2411.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 800;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused Font

File: fonts.css [hunk 11]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-cheltenham';
+  src:
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-italic-300.5278551441f9d1ce9cd247421e26a143.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-italic-300.55c933f230696eae11a43a9a2995b24d.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-italic-300.574eeb5bb2bf407111d0789ef94a6504.ttf') format('truetype');
+  font-style: italic;
+  font-weight: 300;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused fonts

File: fonts.css [hunk 12]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-cheltenham';
+  src:
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-italic-500.c149a85a3db3ccc3cd9e1e217a1e8e4c.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-italic-500.b0a9f31ef2ab8b3f06024843e8b463cf.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/cheltenham/cheltenham-italic-500.0037bc2b5f7715117712e6e1b21d2ae4.ttf') format('truetype');
+  font-style: italic;
+  font-weight: 500;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: delete unused fonts

File: fonts.css [hunk 13]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-cheltenham-small';
+  src:
+    url('https://nytimes.com/fonts/family/cheltenham-small/cheltenham-small-italic-700.4d88cc14057ed010aa281da2202f928c.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/cheltenham-small/cheltenham-small-italic-700.af2a0cfe9b9297ce4a6d0231d2ec85f1.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/cheltenham-small/cheltenham-small-italic-700.47c0a5ef660342477afdbaf083b93caf.ttf') format('truetype');
+  font-style: italic;
+  font-weight: 700;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused fonts

File: fonts.css [hunk 14]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-cheltenham-small';
+  src:
+    url('https://nytimes.com/fonts/family/cheltenham-small/cheltenham-small-normal-700.1a0b316424cdebd18086b8dbbc768eef.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/cheltenham-small/cheltenham-small-normal-700.eccd7f6517b300fee245af2242b63837.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/cheltenham-small/cheltenham-small-normal-700.c7e511fa120d6691a4a7f58ab8d0a593.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 700;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused font

File: fonts.css [hunk 15]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-franklin';
+  src:
+    url('https://nytimes.com/fonts/family/franklin/franklin-italic-600.e45166ea8a537cee27368f60b3c15bb3.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/franklin/franklin-italic-600.7c6a93c684714e03fc6a085eb06110a2.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/franklin/franklin-italic-600.806fcb2d2b3a99b15e96596046ea8f92.ttf') format('truetype');
+  font-style: italic;
+  font-weight: 600;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: delete unused fonts

File: fonts.css [hunk 16]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-franklin';
+  src:
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-500.d6c06a3d84a57100edad5bf9b84ff739.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-500.414758de098dc644b688afa031d35f9b.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-500.1b6dc8a3d86d2a28b825482c9497a633.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused fonts

File: fonts.css [hunk 17]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-franklin';
+  src:
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-600.abe1b34d5a429f8e034860c86c483446.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-600.8a86998af9a6f5487758d78fef54920d.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-600.d274c9e68918ea101e79552dc9da086d.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 600;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused font

File: fonts.css [hunk 18]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-franklin';
+  src:
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-900.c7663abf5510617900d31a082a9c2e8d.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-900.a872cf7b59306d8146237e98fb6be2fa.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/franklin/franklin-normal-900.2c951d5351ef934b47077105a883d2d1.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 900;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused fonts

File: fonts.css [hunk 19]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-franklin-tv';
+  src:
+    url('https://nytimes.com/fonts/family/franklin-tv/franklin-tv-normal-700.9f5d9a89231f67aa612fb6afecd764a6.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/franklin-tv/franklin-tv-normal-700.7458a425ba50c5ac25ca368f74d79c82.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/franklin-tv/franklin-tv-normal-700.8e62e074b39e27ea22ed9f0a40948697.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 700;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused fonts

File: fonts.css [hunk 20]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-graphik-xcond';
+  src:
+    url('https://nytimes.com/fonts/family/graphik-xcond/graphik-xcond-normal-600.307b244de785a412768c48449e1f4ed4.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/graphik-xcond/graphik-xcond-normal-600.09dc367ad0694df886c2f743990f21d5.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/graphik-xcond/graphik-xcond-normal-600.5c5f8eca64b39c7e75b1ed61508c3980.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 600;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused fonts

File: fonts.css [hunk 21]
```diff
@@ -0,0 +1,1462 @@
...
+@font-face {
+  font-family: 'nyt-kippenberger-poster';
+  src:
+    url('https://nytimes.com/fonts/family/kippenberger-poster/kippenberger-poster-normal-900.ccadc5b8ad4bb1689d99cd644f3b5334.woff2') format('woff2'),
+    url('https://nytimes.com/fonts/family/kippenberger-poster/kippenberger-poster-normal-900.4ec00a6de00cfa1b04fe4de3cfa5619e.woff') format('woff'),
+    url('https://nytimes.com/fonts/family/kippenberger-poster/kippenberger-poster-normal-900.b91522150800ca67f5d4d72c37bf0a18.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 900;
+  font-display: swap;
+}
```
Comment (euqueme) [inline fonts.css]: Delete unused fonts

File: fonts.css [hunk 22]
```diff
@@ -0,0 +1,1462 @@
...
+    url('https://nytimes.com/fonts/family/schnyder-scond/schnyder-scond-normal-700.d45b09d2e222c1e8729e995a8f3f1400.ttf') format('truetype');
+  font-style: normal;
+  font-weight: 700;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'nyt-stymie';
+  src:
+    url('https://nytimes.com/fonts/fa
```
Comment (euqueme) [inline fonts.css]: Delete Unused fonts

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE) :woman_technologist:

### Hello Team! 

## Great work so far @kotoku-code and @Spayco, great attention to detail :microscope:  the page looks very similar to the original :medal_sports:   there's still some work to do before we can give you the approval :handshake: 

**IMPLEMENTATION DETAILS**

**Footer**

- [x]  It should have a fixed position (it sticks to the bottom part of the screen)

image

**ADDITIONAL MICROVERSE REQUIREMENTS (Pathwright)**

**General**

- [x] Files not related to the project should not be committed and added to .gitignore (your node_modules folder is not used) 

image

- [x] Comments: It’s better to delete the commented code. If you need to review a previous version of your code, use git history 

- [x] Linters config: your linters aren't working, perhaps you need to activate the repo in [stickler](https://stickler-ci.com/)

**Section specific**

beware of this CSS bad practices:

- [x] no inefficient selectors (remove unused fonts from your font.css file)
- [x] no inline styling (remove all styling from your HTML file they should be defined in your CSS file)

I have commented on the code where lines should be modified or removed :arrow_down_small:

### That's it team :raised_hands: make these mandatory changes :memo: and submit another code review request :rocket:

STATUS OF THE PROJECT: REQUESTED CHANGES :traffic_light:

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE) :woman_technologist:

### Hello again @kotoku-code and @Spayco :wave: 

#### Thank you for doing all the requested changes, I believe your project looks even more professional than it was before :business_suit_levitating:  just one thing:

1. The Live demo is not showing the changes please update it.

#### That's it. Congratulations on this project :star: make this change :memo: and submit the project no need to request for a code review :robot: 

STATUS OF THE PROJECT: APPROVED :green_apple:

**Pull request merged**